src/main/java/com/amihaiemil/docker/PayloadOf.java

```diff
@@ -37,15 +38,17 @@
 import javax.json.JsonValue;
 import org.apache.http.HttpEntityEnclosingRequest;
 import org.apache.http.HttpRequest;
+import org.apache.http.HttpResponse;
 
 /**
  * JSON payload of an HttpRequest.
  *
  * @author George Aristy (george.aristy@gmail.com)
  * @version $Id$
  * @since 0.0.1
+ * @todo #108:30min Add tests for PayloadOf.
  */
-public final class PayloadOf implements JsonObject {
+final class PayloadOf implements JsonObject {
```

Comment:
@llorllale ``PayloadOf`` should extend [com.amihaiemil.docker.JsonResource](https://github.com/amihaiemil/docker-java-api/blob/e526a7495618ca67ce2fba2b56b3f8f9956ec9ab/src/main/java/com/amihaiemil/docker/JsonResource.java) which is basically an abstract JsonObject (also extended by RtContainer, RtImage etc).

Reply:
@amihaiemil right